src/kinetics/Reaction.cpp

```diff
@@ -385,6 +385,52 @@ ElectrochemicalReaction::ElectrochemicalReaction(const Composition& reactants_,
 {
 }
 
+BlowersMaselReaction::BlowersMaselReaction()
+    : Reaction()
+    , allow_negative_pre_exponential_factor(false)
+{
```

Comment:
Explicitly calling the default constructor is not necessary.
```suggestion
BlowersMaselReaction::BlowersMaselReaction()
    : allow_negative_pre_exponential_factor(false)
{
```